todo-backend/README.md

```diff
@@ -0,0 +1,67 @@
+# Todo-Backend
+
+[Todo-Backend](https://www.todobackend.com/) is a typical example to showcase backend tech stacks. It defines a simple web API for managing a Todo List.
+
+This sample is an implementation of a Todo-Backend with Binaris functions.
+
+# How does it work?
+
+1. A public function exposes REST API to the outside world
+1. A private function uses a Redis Cache instance for the data storage purpose
+
+# Deploy me!
+
+## Get some API Keys
+1. [Binaris API Key](https://binaris.com/)
+1. A Redis instance. A free, managed one is available from [Redis Labs](www.redislabs.com)
+
+## Deploy
+```bash
+# Install the Binaris CLI
+$ npm install -g binaris
+
+# Set your API key in ~/.binaris.yml
+$ bn login
+
+# Clone this repo
+$ git clone git@github.com:binaris/functions-examples.git
+$ cd functions-examples/todo-backend
+
+# Set the environment variables for deployment
+$ export BINARIS_ACCOUNT_ID={your-binaris-account-id}
+$ export BINARIS_API_KEY={your-binaris-api-key}
+$ export REDIS_HOST={your-redis-host}
+$ export REDIS_HOST={your-redis-port}
```

Comment:
you have REDIS_HOST twice, needs to be REDIS_PORT

Reply:
Sorry, I wasn't careful enough with the README.